packages/elements/src/photon-multirx-form/photon-prescribe-workflow.tsx

```diff
@@ -166,6 +189,53 @@ export function PrescribeWorkflow(props: PrescribeProps) {
     ref?.dispatchEvent(event);
   };
 
+  // let's start screening all of the prescriptions we're drafting
+  const screenDraftedPrescriptions = async () => {
+    // start out by getting the treatment id of the prescription we're drafting now -
+    // we'll want to knwo about it so we cn show an alert underneath it, before it gets
+    // added to the order
+    const inProgressDraftedPrescriptionTreatmentId = props.formStore.treatment?.value?.id;
+
+    // and then get the ones already added to the order (but not persisted)
+    const draftedPrescriptions = [...props.formStore.draftPrescriptions.value];
+
+    // if there is one, add in the prescription being created
+    if (inProgressDraftedPrescriptionTreatmentId) {
+      draftedPrescriptions.push({
+        treatment: { id: inProgressDraftedPrescriptionTreatmentId }
+      });
+    }
+
+    // pluck out all the attributes we won't need so we can make a screening request
+    const sanitizedDraftedPrescriptions = draftedPrescriptions.map(
```

Comment:
Im pretty sure Apollo will do this for you

Reply:
yeah great point let me evaluate to see if i even need this logic

Reply:
note: it doesn't and if you give it stuff it's not prepared for it will blow up. not removing these will cause it to blow up.